config/imsim-config-instcat.yaml

```diff
@@ -9,369 +9,30 @@
-                # TODO: Figure out the depth to use for other bands.  Josh found -0.6 for y.
-                # These numbers are in units of pixels.
-                ddepth_dict: {'u':0, 'g':0, 'r':0, 'i':0, 'z':0, 'y':-0.6}
-                sband: { type: OpsimData, field: band }
-        -
-            type: Refraction
-            index_ratio: 3.9  # TODO: This is what Josh used for y band.
-                              # I assume it's wavelength dependent...
-                              # Probably need to use the same kind of pattern as above for depth.
-
-    world_pos:
-        type: InstCatWorldPos
-
-# This defines both the output files and some basic things about the overall exposure/fov.
-output:
-    type: LSST_CCD
-    nproc: 1    # Change this to work on multiple CCDs at once.
-    nfiles: 1   # Default is all 189 CCDs.  Set to 1 while testing.
-
-    camera: LsstCam
-
-    exptime: $exptime
-
-    cosmic_ray_rate: 0.2
-
-    det_num:
-        type: Sequence
-        nitems: 189
-        first: 94  # Can set first to something if you want to do a specific sensor.
-
-    dir: fits
-    file_name:
-        type: FormattedStr
-        format : eimage_%08d-%1d-%s-%s-det%03d.fits
-        items:
-            - { type: OpsimData, field: observationId }
-            - { type: OpsimData, field: snap }
-            - { type: OpsimData, field: band }
-            - "$det_name"   # A value stored in the dict by LSST_CCD
-            - "@output.det_num"
-
-    readout:
-        # Convert from e-image to realized amp images
-        readout_time: 3.
-        dark_current: 0.02
-        bias_level: 1000.
-        pcti: 1.e-6
-        scti: 1.e-6
-        bias_levels_file: LSSTCam_bias_levels_run_13421.json
-
-        file_name:
-            type: FormattedStr
-            format : amp_%08d-%1d-%s-%s-det%03d.fits.fz
-            items:
-                - { type: OpsimData, field: observationId }
-                - { type: OpsimData, field: snap }
-                - { type: OpsimData, field: band }
-                - "$det_name"
-                - "@output.det_num"
-
-    truth:
-        dir: output
-        file_name:
-            type: FormattedStr
-            format : centroid_%08d-%1d-%s-%s-det%03d.txt.gz
-            items:
-                - { type: OpsimData, field: observationId }
-                - { type: OpsimData, field: snap }
-                - { type: OpsimData, field: band }
-                - "$det_name"
-                - "@output.det_num"
-        columns:
-            object_id: "@object_id"
-            ra: "$sky_pos.ra.deg"
-            dec: "$sky_pos.dec.deg"
-            x: "$image_pos.x"
-            y: "$image_pos.y"
-            # TODO: ... more probably
-
-gal:
-    type: InstCatObj
+# Get most of the configuration from the imSim config-template
+# for instance catalogs.
+template: imsim-config
 
 ################################################################
 #  Above is template material.
```

Comment:
I guess this is still true.  But since it's one line above now, this comment should probably be amended.

Reply:
OK I worked on the text in both files.